visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.spec.ts

```diff
@@ -80,28 +84,27 @@ describe("screenshotButtonComponent", () => {
 		const clickDownloadLinkSpy = jest.spyOn(HTMLAnchorElement.prototype, "click").mockImplementation()
 		const makeScreenshotToFileSpy = jest.spyOn(fixture.componentInstance, "makeScreenshotToFile")
 
-		await user.click(screen.getByTitle("Take a screenshot of the map with Ctrl+Alt+S (save as file) or Ctrl+Alt+F (copy to clipboard)"))
+		await fixture.componentInstance.makeScreenshotToFile()
 
+		expect(makeScreenshotToFileSpy).toHaveBeenCalledTimes(1)
 		await waitFor(() => {
-			expect(makeScreenshotToFileSpy).toHaveBeenCalledTimes(1)
 			expect(clickDownloadLinkSpy).toHaveBeenCalledTimes(1)
 		})
 	})
 
 	it("should be disabled when write to clipoard is not available in browser", async () => {
-		const user = userEvent.setup()
+		;(checkWriteToClipboardAllowed as jest.Mock).mockImplementation(() => false)
```

Comment:
remove unnecessary semicolon

Reply:
This semicolon is needed to prevent concatenation errors.